gettsim/benefits/arbeitsl_geld_2.py

```diff
@@ -9,8 +15,14 @@ def alg2(household, params):
         parents. Income and wealth is tested for, the transfer withdrawal rate is
         non-constant.
     """
+    if params["jahr"] < 2005:
+        # warnings.warn("Arbeitslosengeld 2 existiert erst ab 2005 und wurde deshalb "
+        #               "nicht berechnet.")
+        return household
```

Comment:
As we initialize all output columns in `tax_transfer.py` with nan, they will just contain nans. I think that is better, because 0 could also be a real value?!

Reply:
Probably a general design issue which is better discussed elsewhere.